Editor/Scripts/DCCIntegration/Integrators/MayaIntegrator.cs

```diff
@@ -1,201 +1,200 @@
-﻿using System;
+using System;
```

Comment:
What are all the changes in this file?

If this is just tidy up work without logic changes can you commit in a separate PR so it's clear where the logic changes are?